custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```
@@ -564,7 +564,7 @@
         },
         {
             "name": "separator_evaluation",
-            "label": "Train / Test Set",
+            "label": "Train / Evaluation Set",
```

To comply with the Visual ML UX, I would say
```suggestion
            "label": "Train / test set for final evaluation",
```